static/rfcs/17/rln-relay.msc

```diff
@@ -1,32 +1,23 @@
 # Sequence diagram for RLN Relay protocol (registration)
 msc {
-  hscale = "2";
+  hscale = "1";
 
   d [label = " "], a [label = " "],b [label = " "];
   a rbox a [label="Relay Node A"], 
   b note b [label="Membership Contract"],
   d rbox d [label = "Relay Node B"];
+ 
   |||;
+  d abox d [ label=" \n Listening to the membership contract \n "] ;
 
   a box a [ label=" \n Generate sk,pk \n "] ;
   a=>b [ label = " \n Register(pk, x ETH) \n " ] ;
```

Comment:
Can the label go on top of the arrow rather over it?

Reply:
Yes, it can, done in https://github.com/vacp2p/rfc/pull/497/commits/f0f4d58cbeabc3c7cf8a09a3a3b967a8155f48a3